src/mixxx.cpp

```diff
@@ -842,6 +842,16 @@ void setVisibilityOptionState(QAction* pAction, ConfigKey key) {
     pAction->setChecked(pVisibilityControl != NULL ? pVisibilityControl->get() > 0.0 : false);
 }
 
+void MixxxMainWindow::toggleCheckedSamplers()
+{
+    disconnect(m_pViewShowSamplers, SIGNAL(toggled(bool)), 
+               this, SLOT(slotViewShowSamplers(bool)));
+    ConfigKey key("[Samplers]", "show_samplers");
+    m_pViewShowSamplers->setChecked(ControlObject::get(key));
```

Comment:
Did you check the behavior, when there is no  "[Samplers]", "show_samplers" instantiated from the skin?
You should add a note hereabout that case as comment in the code.